app/assets/stylesheets/modules/footer.scss

```diff
@@ -0,0 +1,44 @@
+@import "variables";
+
+.footer {
+
+  background: $blue;
+  height: 300px;
+  color: $white;
+
+  .header {
+    font-family: Lato;
+    padding-top: 20px;
+    text-align: center;
+  }
+
+  h2 {
+    font-family: Lato;
+    line-height: 1.4;
+    color: white;
+  }
+
+  hr {
+    border: solid #DDDDDD;
+    border-width: 1px 0 0;
+    clear: both;
+    margin: 1.25rem 0 1.1875rem;
```

Comment:
Properties should be ordered border, border-width, clear, height, margin, margin-left, margin-right